### PR TITLE
Clean generated files in ObjC2Winmd projections projects

### PR DESCRIPTION
**Pull request by memontic-ms**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc/1355)

### CODE REVIEW

**Comment by asimonov-msft** [comment]
:shipit:

File: msvc/sbclang.targets [hunk 1]
```diff
@@ -273,6 +273,27 @@
...
+    Condition="'$(IslandwoodConfigurationType)' == 'Projection' And Exists('$(Objc2WinmdGeneratedFilesDir)')">
+
+    <!-- Clean the files generated by objc2winmd from the project. -->
+    <ItemGroup>
+      <Objc2WinmdGeneratedFiles Include="$(Objc2WinmdGeneratedFilesDir)\*.mm" />
+      <Objc2WinmdGeneratedFiles Include="$(Objc2WinmdGeneratedFilesDir)\*.idl" />
+      <Objc2WinmdGeneratedFiles Include="$(Objc2WinmdGeneratedFilesDir)\*.h" />
+      <!-- Include dlldata.c file that was generated by MIDL -->
+      <Objc2WinmdGeneratedFiles Include="$(Objc2WinmdGeneratedFilesDir)\dlldata.c" />
+    </ItemGroup>
```
Comment (pradipd) [inline msvc/sbclang.targets]: What about the InvokeObjc2Winmd.dummy file?

Comment (memontic-ms) [inline msvc/sbclang.targets]: Some other task is deleting the InvokeObjc2Winmd.dummy on clean

**Comment by DHowett-MSFT** [comment]
:shipit:

**Comment by pradipd** [comment]
:shipit:

**Pull request merged**